tools/webpack/packages.js

```diff
@@ -29,6 +29,7 @@ const BUNDLED_PACKAGES = [
 	'@wordpress/interface',
 	'@wordpress/undo-manager',
 	'@wordpress/sync',
+	'@wordpress/dataviews',
```

Comment:
I think there's another place where we declare the bundled packages. Maybe the webpack dependency extraction plugin

Reply:
Good catch! I'll update